### ISSUE TITLE
Empty resumption token

### ISSUE THREAD

**Issue by Xennis** [mloesch/sickle#8]
**Error**
When I request the last entries of CiteSeer (for now it is http://citeseerx.ist.psu.edu/oai2?verb=ListRecords&metadataPrefix=oai_dc&from=2015-12-07), I get the error:

```
sickle.oaiexceptions.BadArgument: metadataPrefix is required
```

**Reason for this error (maybe)**
Since I receive this error after I read all entries, I think the problem could be the following:

CiteSeer returns an empty `<resumptionToken/>` element, which is correct according to the [OAI PHM documentation](http://www.openarchives.org/OAI/openarchivesprotocol.html#FlowControl):
> the response containing the incomplete list that completes the list must include an **empty resumptionToken element**;

But in https://github.com/mloesch/sickle/blob/master/sickle/iterator.py#L61 you only check, if the element exists and not if also the text of this element exists.

**Comment by mloesch** [comment]
Please make sure you are using the latest version, this has been fixed in 0.5.

**Comment by Xennis** [comment]
Okay, thanks.
